hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OzoneClientKeyGenerator.java

```diff
@@ -93,6 +103,9 @@ public Void call() throws Exception {
 
     contentGenerator = new ContentGenerator(keySize, bufferSize);
     metadata = new HashMap<>();
+    replicationConfig = ReplicationConfig
```

Comment:
This line will conflict with [the patch of EC](https://github.com/apache/ozone/pull/2996/files#diff-28399319a85aa26781a7ab36e4ae8232b086656016e76071edad5ed8b3b6430eR117). EC is currently voting on merge to master, so EC will merge into master before streaming.  So can we continue this PR after EC merge?

Reply:
Sure.

Reply:
@sadanand48 , let's hardcode replicationConfig in createStreamKey(..) for the moment.  We may change it later if there is a need.
```
  private void createStreamKey(long counter) throws Exception {
    final ReplicationConfig replicationConfig = ReplicationConfig
        .fromProtoTypeAndFactor(HddsProtos.ReplicationType.RATIS,
            HddsProtos.ReplicationFactor.THREE);
    final String key = generateObjectName(counter);

    timer.time(() -> {
      try (OzoneDataStreamOutput stream = bucket
          .createStreamKey(key, keySize, replicationConfig, metadata)) {
        contentGenerator.write(stream);
      }
      return null;
    });
  }
```